README.md

```diff
@@ -101,6 +101,12 @@ should be installed as one of your project's `devDependencies`:
 ```
 npm install --save-dev @testing-library/react
 ```
+or
+
+This module is distributed via [yarn](https://classic.yarnpkg.com/en/)
+```
+yarn add --dev jest @testing-library/react @testing-library/user-event jest-dom nock
```

Comment:
I would move this to the npm block above so that it's clearer that these commands do the same thing. It should also install the same dependencies. `user-event`, `jest-dom` and `nock` are not required to use this library.

Reply:
Sure, my only idea behind this PR was to add the yarn installation method as well to make the user stick to this particular repo and docs rather than making random search over web.

Reply:
It's still unclear why you think `@testing-library/user-event jest-dom nock` are required for usage with `yarn`. I use none of these libraries and `yarn` and hadn't had any issue with missing dependencies.

Reply:
Oh okay so, what would be added for users who are using ```yarn``` and need a solution for it , because yarn add --save-dev @testing-library/react seems okay to me, let me update the changes. 

Sorry, but I followed an online article that uses the previous command that updated for yarn installation.